### ISSUE TITLE
how to inference

### ISSUE THREAD

**Issue by syha2994** [nelson1425/EfficientAD#7]
hello. thanks for the nice code

How do I just infer an image?

**Comment by kywish** [comment]
U can refer to the [test ](https://github.com/nelson1425/EfficientAD/blob/ce0275cbe1be590a5dbf18959db430ffdaf98561/efficientad.py#LL262C5-L262C5)function in efficientad.py

**Comment by JHC521PJJ** [comment]
map_combined, map_st, map_ae = predict(input_img, teacher_model, student_model, 
                                               ae_model, teacher_mean, teacher_std,
                                               q_st_start=q_st_start, q_st_end=q_st_end, 
                                               q_ae_start=q_ae_start, q_ae_end=q_ae_end)
        
        map_combined = torch.nn.functional.pad(map_combined, (4, 4, 4, 4))
        map_combined = torch.nn.functional.interpolate(
        map_combined, (orig_height, orig_width), mode='bilinear')
        map_combined = map_combined[0, 0].cpu().detach().numpy()
        ad_score = np.max(map_combined)

the ad_score is your need.
